packages/app/src/cli/services/app/config/link.test.ts

```diff
@@ -560,6 +560,80 @@ embedded = false
     })
   })
 
+  test('fetches the privacy compliance webhooks from the configuration module', async () => {
+    await inTemporaryDirectory(async (tmp) => {
+      // Given
+      const options: LinkOptions = {
+        directory: tmp,
+        developerPlatformClient,
+      }
+
+      vi.mocked(loadApp).mockRejectedValue('App not found')
+      vi.mocked(fetchOrCreateOrganizationApp).mockResolvedValue(mockRemoteApp())
```

Comment:
had to remove the prev `vi.mocked(fetchAppExtensionRegistrations).mockResolvedValue(...)` call bc it keeps saying the function doesn't exist :/ I think it's due to some changes in the `link.ts` file